js/UrbanAirship.js

```diff
@@ -494,6 +494,32 @@ class UrbanAirship {
     return UrbanAirshipModule.isQuietTimeEnabled();
   }
 
+  /**
+   * Sets Autobadging on iOS, either to true or false. Badging is not supported for Android.
+   * 
+   * @param {boolean} enabled Whether or not to enable Autobadging.
+   */
+  static setAutobadgeEnabled(enabled: boolean) {
+    if (Platform.OS == 'ios') {
+      UrbanAirshipModule.setAutobadgeEnabled(enabled);
+    } else {
+      console.log("This feature is not supported on this platform.")
+    }
+}
+
+  /**
+   * Checks to see if Autobadging on iOS is enabled. Badging is not supported for Android.
+   * 
+   * @return {Promise.<boolean>} A promise with the result, either true or false.
+   */
+  static isAutobadgeEnabled(): Promise<boolean> {
+    if (Platform.OS == 'ios') {
+      return UrbanAirshipModule.isAutobadgeEnabled();
+    } else {
+      console.log("This feature is not supported on this platform.")
```

Comment:
what happens if someone calls this on android, will it return undefined? if so do we want it to return false?

Reply:
I can certainly add that, or maybe return null since there's nothing to really do if it's android?

Reply:
I'd suggest returning false. since the method is still technically callable from the JS side on android, that will make its return signature consistent, and "unavailable" just means "permanently disabled"

Reply:
Can do!